web/src/recent_view_ui.ts

```diff
@@ -715,6 +715,9 @@ function format_conversation(conversation_data: ConversationData): ConversationC
             unread.num_unread_mentions_for_user_ids_strings(user_ids_string) > 0;
 
         let is_bot = false;
+        const is_deactivated = !people.is_active_user_for_popover(
+            Number.parseInt(user_ids_string, 10) || 0,
+        );
```

Comment:
Similarly this should be inside the `!is_group` block, since we only have a single user in that case.